payments-core/res/drawable/stripe_ic_paymentsheet_cvc.xml

```diff
@@ -1,23 +1,18 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think this changed it to the wrong icon.  I think it is now the discover icon?